client/state/data-layer/wpcom/index.js

```diff
@@ -14,6 +14,7 @@ import concierge from './concierge';
 import domains from './domains';
 import gravatarUpload from './gravatar-upload';
 import i18n from './i18n';
+import jetpackConnectionOwner from './jetpack/connection/owner';
```

Comment:
Currently, the existing Jetpack connection redux implementation uses thunks, but we will probably end up refactoring it to data layer sooner or later. To prepare for that, I think it would be nice to hook `./jetpack/connection` here, and then have it take the responsibility to merge handlers for `./owner`, `./` and `./data` (these two currently use thunks). What do you think?